.github/workflows/auto-version.yml

```diff
@@ -0,0 +1,34 @@
+name: Increment Version
+
+permissions:
+  contents: write
+
+on:
+  workflow_call:
+
+jobs:
+  increment-version:
+    name: Increment Version
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout codebase
+        uses: actions/checkout@v4.1.1
+        with:
+          fetch-depth: 0
+
+      - name: Get versioning config files
+        run: |
+          config_path=https://raw.githubusercontent.com/health-education-england/.github/main/.github/workflows/auto-version-config
+          curl $config_path/.versionrc > .versionrc
+          curl $config_path/terraform-version-updater.js > terraform-version-updater.js
```

Comment:
Feels a bit awkward, but it's a simple enough way for the workflow to provide it's own configuration rather than have to copy paste everywhere (and avoids the maintenance overhead that comes with it).